Src/coffeescript/cql-execution/src/elm/quantity.coffee

```diff
@@ -103,7 +106,7 @@ get_ucum_units = (units) ->
   else units
 
 # The smallest common duration is the millisecond
```

Comment:
I still think the comment should explain what is actually being done in the function. But if no one else thinks its an issue, ill approve this.